Packs/Phishing/ReleaseNotes/3_0_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Phishing - Generic v3
+- %%UPDATE_RN%%
```

Comment:
Description is missing. Also, the versio should be changed in the pack metadata file.

Reply:
lol for some reason the changes weren't pushed..
I <3 VS Code